Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -117,17 +131,20 @@ public class Client: CustomStringConvertible {
                   stateStorage: stateStorage,
                   httpClient: chttpClient,
                   jwks: jwks,
-                  tokenHandler: tokenHandler)
+                  tokenHandler: tokenHandler,
+                  tracker: tracker)
     }
     
-    init(configuration: ClientConfiguration, sessionStorage: SessionStorage, stateStorage: StateStorage, httpClient: HTTPClient, jwks: JWKS, tokenHandler: TokenHandler) {
+    init(configuration: ClientConfiguration, sessionStorage: SessionStorage, stateStorage: StateStorage, httpClient: HTTPClient, jwks: JWKS, tokenHandler: TokenHandler, tracker: TrackingEventsHandler? = nil) {
```

Comment:
nit: very long line, can you break it down like you did above?

Reply:
Good point!

(Hope there will be a time in the future to add SwiftLint or swift-format to the project)